src/test/java/uk/gov/pay/directdebit/payments/resources/PaymentViewResourceITest.java

```diff
@@ -341,15 +308,15 @@ public void shouldReturn6Records_whenAmountSet() {
                     .insert(testContext.getJdbi());
         }
 
-        String requestPath = "/v1/api/accounts/{accountId}/payment-requests/view?amount=:amount"
+        String requestPath = "/v1/api/accounts/{accountId}/transactions/view?amount=:amount"
                 .replace("{accountId}", testGatewayAccount.getExternalId())
                 .replace(":amount", "2343");
         givenSetup()
                 .get(requestPath)
                 .then()
                 .statusCode(Response.Status.OK.getStatusCode())
                 .contentType(JSON)
-                .body("payment_views", hasSize(6));
+                .body("results", hasSize(6));
     }
 
     private RequestSpecification givenSetup() {
```

Comment:
think you need a test for mandate id in here too?

Reply:
Mandate id is not included in the response json object